skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="<?= $lang ?>" dir="<?= $dir ?>">
+<html lang="<?= $lang ?>" dir="<?= $dir ?>" class="<?= implode( ' ', $htmlClasses ) ?>">
```

Comment:
Just in case someone accidentally uses `addHtmlClass` with user supplied data in the future, it might be worth making this:
```php
<?= implode( ' ', array_map( 'Sanitizer::escapeClass', $htmlClasses ) ) ?>
```